Dfe.Academies.Academisation.IService/ServiceModels/Legacy/ProjectAggregate/InvoluntaryProjectServiceModel.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace Dfe.Academies.Academisation.IService.ServiceModels.Legacy.ProjectAggregate
+{
+	public class InvoluntaryProjectServiceModel
+	{
+		public SchoolServiceModel? School { get; init; }
+		public TrustServiceModel? Trust { get; init; }
+	}
+
+	public class TrustServiceModel
```

Comment:
Should we clarify that this is only for InvoluntaryConversions? This TrustServiceModel won't be used for anything other than it. Perhaps InvoluntaryProjectTrustServiceModel or something similar?

Reply:
Yea makes sense I did that in the UI, not sure why its different here.